package.json

```diff
@@ -42,19 +44,19 @@
     "eslint-plugin-react": "^7.0.0",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.0",
-    "http-server": "^0.9.0",
+    "http-server": "^0.11.1",
     "json-loader": "^0.5.4",
     "less": "^2.7.1",
     "less-loader": "^4.0.5",
-    "parallelshell": "^2.0.0",
+    "npm-run-all": "^4.1.3",
     "style-loader": "^0.18.2",
     "titlecaps": "^0.3.2",
     "uglifyjs-webpack-plugin": "^1.0.1",
     "url-loader": "^0.5.6",
-    "webdriver-manager": "^11.1.0",
-    "webdriverio": "^4.5.2",
-    "webpack": "^3.6.0",
-    "webpack-dev-server": "^2.8.2"
+    "webdriver-manager": "^12.0.6",
+    "webdriverio": "^4.12.0",
+    "webpack": "^3.12.0",
+    "webpack-dev-server": "^2.11.2"
```

Comment:
`webpack` & `webpack-dev-server` shouldn't be related to the test?

the other package upgrade shouldn't affect the tests either (or is it?), but let's keep the others, as it is related to the test.

Reply:
I can't merge anyway, needs repush because Netlify breaks.

Reply:
The build was not fast enough.

Reply:
`parallelshell` and `webdriver` affect the test. `webpack` and `httpserver` don't affect, but they are related to the test, so I also upgrade them.

`webpack` is used here https://github.com/coala/gh-board/pull/42/commits/cc9a84b9c4be1067caacd42ec138fb0ada315e4a#diff-5d545609ae4b281a1f2100c2d8ea622eL2